Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -55,8 +58,21 @@ public struct PreplannedListItemView: View {
     @ViewBuilder private var downloadButton: some View {
         switch model.status {
         case .downloaded:
-            Image(systemName: "checkmark.circle")
-                .foregroundStyle(.secondary)
+            Button {
+                Task {
+                    if let map = await model.loadMobileMapPackage() {
+                        onMapSelectionChanged(map)
+                    }
+                }
+            } label: {
+                Text("Open")
+                    .font(.system(.headline, weight: .bold))
+                    .foregroundStyle(Color.accentColor)
+                    .frame(width: 76, height: 32)
+                    .background(Color(.tertiarySystemFill))
+                    .clipShape(.rect(cornerRadius: 16))
+            }
+            .buttonStyle(.plain)
```

Comment:
A few things in here that I'd like to see corrected:
- headline font is probably too large
- We should really try to refrain from using explicit frame sizes. Since we are using dynamic fonts, using hard-coded sizes will not scale properly.
- If you want a capsule button, there is a way to do that with the system. Don't need to set the bg color and clip shape manually.

This should do the trick:

```swift
                Button {
                     ....
                } label: {
                    Text("Open")
                        .fontWeight(.bold)
                }
                .buttonStyle(BorderedButtonStyle())
                .buttonBorderShape(.capsule)
```

Reply:
I agree, fixed frame can be changed. The reason a clipped rectangle instead of a capsule is used, is that it better mimics the App Store "Open" button. The capsule button looks too wide when used here.

Reply:
FYI: For the button style modifier, use `bordered`: `.buttonStyle(.bordered)`.

Reply:
Here is a screenshot of the updated UI with these suggestions applied:

<img src="https://github.com/user-attachments/assets/66ae3717-fd6b-45e7-8729-41bcab9162c1" width="300">

Reply:
I confess that I forgot to actually run the code to see what the UI looks like. That refresh button (I'm assuming that's what it is) is non-standard. Did you try implementing [pull to refresh](https://www.hackingwithswift.com/quick-start/swiftui/how-to-enable-pull-to-refresh)?

Reply:
Should pull-to-refresh apply to a modally presented sheet? I feel the pull down gesture will interfere with the dismiss action for this view. 🤔

Reply:
Pull to refresh is perfectly valid for a sheet, as long as pull to dismiss is disabled. I know that UIKit had a way of doing that. I assume that SwiftUI just does the right thing, but I don't know for certain.

Reply:
```swift
        .interactiveDismissDisabled()
```

Reply:
The Calendars view in the Calendar app is a sheet with pull to refresh, for example.

Reply:
@rolson for the win!

Reply:
I will log an issue and implement pull to refresh in another PR.